aibtc-v1/utils/crews.py

```diff
@@ -7,6 +7,7 @@
 class AIBTC_Crew:
     def __init__(self, name: str):
         self.name = name
+        self.description = ""
```

Comment:
Maybe the same for description where we pass it on init, so it can live in the crew file like this:

```python
class WalletSummaryCrew(AIBTC_Crew):
    def __init__(self):
        super().__init__(
            "Wallet Summarizer",
            "Description of Wallet Summarizer",
            embedding session_state
        )
```